lib/tf_lock/tf_lock_acquire.py

```diff
@@ -43,6 +46,31 @@ def tf_lock_acquire(root_module: Path) -> ExitCode:
                 )
                 return 0
 
+            else:
```

Comment:
remove this `else`, since the rest of this function is using the early return pattern